tests/unit/samplegen/test_template.py

```diff
@@ -301,6 +312,226 @@ def test_dispatch_map_loop():
     )
 
 
+def test_print_input_params():
+    check_template(
+        '''
+        {% import "feature_fragments.j2" as frags %}
+        {{ frags.printInputParams([{"base": "squid",
+                                    "body": [{"field": "mass",
```

Comment:
Do you think it would be useful to have an input variable with an underscore to test casing?

Reply:
The `input_parameter` in `test_render_attr_file` has underscores.